### PR TITLE
Switch to more up-to-date Vagrant PPA

### PR DESCRIPTION
**Pull request by Peaches491**

### CODE REVIEW

File: setup-ubuntu/roles/vagrant/tasks/main.yml [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+  apt_repository:
+    repo: deb https://vagrant-deb.linestarve.com/ any main
+  become: yes
+
+- name: Install Vagrant apt package
   apt:
-    pkg:
-    - vagrant
+    name: vagrant
+    update_cache: true
```
Comment (chpatton013) [inline setup-ubuntu/roles/vagrant/tasks/main.yml:17]: Why did you need to specify this? Shouldn't the apt module figure this out itself by you using the `apt_repository` module to make database changes?

File: setup-ubuntu/roles/vagrant/tasks/main.yml [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
   become: yes
+
+# See: https://computingforgeeks.com/install-latest-vagrant-on-ubuntu-debian-kali-linux/
```
Comment (chpatton013) [inline setup-ubuntu/roles/vagrant/tasks/main.yml:20]: Would you mind adding a reference to this issue as well? (https://github.com/hashicorp/vagrant-installers/issues/12)
This is the saga of why HashiCorp can't be bothered to provide an official PPA.

Also, could you add a TODO about installing Vagrant through their recommended method: https://github.com/hashicorp/vagrant/issues/7529#issuecomment-229216232

File: setup-ubuntu/roles/vagrant/tasks/main.yml (hunk 1 above)
Comment (Peaches491) [reply setup-ubuntu/roles/vagrant/tasks/main.yml:17]: Yep, you're right. Per the ansible docs, `apt_repository` will do this automatically: 
https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html#parameter-update_cache

Removed.

File: setup-ubuntu/roles/vagrant/tasks/main.yml (hunk 2 above)
Comment (Peaches491) [reply setup-ubuntu/roles/vagrant/tasks/main.yml:20]: Done and done
